layouts/partials/footer.hbs

```diff
@@ -13,3 +13,23 @@
         </div>
     </div>
 </footer>
+<script type="text/javascript">
+  (function(d) {
+    try {
+      var xhr = new XMLHttpRequest();
+      xhr.addEventListener('load', function() {
+        if(this.responseText) {
+          addStyle(this.responseText);
+          window.localStorage.fonts_css_cache = this.responseText;
+        }
+      });
+      xhr.open('get', '/static/fonts/webfont.css');
+      xhr.send();
```

Comment:
btw.: yes, we load the webfonts.css every time, and also update the cache every time. The user still gets a fast experience, the webfonts.css will be cached, but in case of new and different fonts, we still get an update.